usaspending_api/awards/v2/filters/location_filter_geocode.py

```diff
@@ -172,11 +176,47 @@ def get_award_ids_by_city(scope: str, city: str, country_code: str, state_code:
     query = {
         "bool": {
             "must": [
-                {"match": {"{}_city_name".format(scope): es_sanitize(city)}},
-                {"match": {"{}_country_code".format(scope): es_sanitize(country_code)}},
+                {"match": {"{}_city_name.keyword".format(scope): es_sanitize(city)}},
             ]
         }
     }
+
+    if country_code != "USA":
+        # A non-USA selected country
+        if country_code != "FOREIGN":
```

Comment:
Is this the main place "FOREIGN" is used? We should move towards putting special sentinel values like this in constants to make our logic more clear and make refactoring easier.